### PR TITLE
fix(token-definitions): unify cardano addresses

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- strip encoded name in cardano `policy id`

## Related Issue

Resolve

### CODE REVIEW

File: suite-common/token-definitions/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
         "jws": "^4.0.0"
     },
     "devDependencies": {
+        "@trezor/blockchain-link-utils": "workspace:*",
```
Comment (AdamSchinzel) [inline suite-common/token-definitions/package.json]: This is on purpose? I think i should be inside `dependencies`

Comment (tomasklim) [reply suite-common/token-definitions/package.json]: Why, it is used in scripts, not in the app. I will move it but there has to be something wrong with configuration

Comment (tomasklim) [reply suite-common/token-definitions/package.json]: it is used in the same place as `ajv` and that dep can be in devDependencies so i dont know

**Review by AdamSchinzel — APPROVED** [review]
LGTM

**Pull request merged**